src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.css">
+  </head>
+  <body>
+    <arcticle class="cart" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="Product"
+        class="cart__image"
+      >
+      <h2 class="cart__title title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="cart__code code">
+        Product code: 195434
+      </p>
+
+      <div class="cart__reviews reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <span class="reviews_number">
+          Reviews: 5
+        </span>
+      </div>
+
+      <div class="cart__price price">
+        <span class="price_name">
+          Price:
+        </span>
+
+        <span class="price_value">
+          $2,199
+        </span>
+      </div>
```

Comment:
```suggestion
      <p class="cart__price price">
        <span class="price_name">
          Price:
        </span>

        <span class="price_value">
          $2,199
        </span>
      </p>
```
now it looks like a paragraph with two strings